File: loops/notes.py
#zoey sosa, 

#What is a loop? 
 #a section of code tthat repeates multiple times

#What are the two types of loops?
    #for loop
nums= [12,3,66,7,3,3,2]

for num in nums:
    print (num)

    #while loop
x=0

while x <10 :
    print(x)
    x+=1

#What is iteration
    # that specific instance of the loop
    #iteration the amount of times you are looping through 

#What are lists? 
nums = [1,2,3,4,5,6,7,6]
siblings = ["alex", "katie", "anderw", "vienna", "tia"]
print(nums)# prints whole list is ugly
print(siblings[3])#prints 1 item from the list

siblings[7] ="jake"
siblings.pop(5)
siblings.insert(1,"jayshree")
siblings.insert(2,["joe", "Noah", "Zee"])
print(siblings)


#How do you make lists in python? 
#1 [] use braces 
#2 add in items with correct data types 
#3 a coma between everything 

#How do you make for loops in python? 
for sibling in siblings:
    print(sibling)

for x in range (1,101, 20):
    print(x)

#How do you make while loops in python?
import random 
x = 1 #varible to keep count of iteration
goose = random.randint(1,20)

while x <=20:
    if x == goose:
    print("goose")
    break # tells the loop to stop 
else :
    print("duck")
    x+=1




#How do you make lists in C?
#How do you make for loops in C?
#How do you make while loops in C?

File: loops/notes.c
//zoey sosa, Notes in C
#include <stdio.h>

int main(void){
 //What is a loop? 
    // a section of code repeated multiple time
//What are the two types of loops?
    //while loops
    int start = 0;
    while(start <5){
        print("hello!\n");
        start++;
    }
    //for loops
    int q;
    for (q=0;q<5;q++){
        print("%d\n",q);
    }
//What is iteration
    //repeating something with minor changes each time 
    // like q becuase it changes everytime
//What are lists? 
    //A array is a varible holding multipe values 

//How do you make arrays (lists) in C?
    //arrays items must be all the same data type
int grades[] = {88, 97, 100, 70, 72, 99, 61};
// 1. set data type
//2. AFTER naming put brackets and write the lenth of the list
//3. list is surrounded by curly brackets {}
//4. commas, between each item

//How do you make for loops in C?
printf("CSP Grade: %d\n", grades[2]);
// change an item in the array
grades[2] = 95;
printf("CSP Grade: %d\n", grades[2]);

// size of list in bytes
int size = sizeof (grades);
//length in list items 
int length = sizeof (grades)/sizeof(grades[0]);
printf("the array is %d items long.\n", length);
// array with strings
//frist brackets sets elngth of list
//seconds brackets sets length of each string
char movies[][20] = {"Cars","treasure plant", "an american tale", "marley and me", "the avengers"};
printf("the movie is %s!\n", movies[1]);
int mlength = sizeof(movies)/sizeof(movies[0]);
//set the iterator, keep track of times through the loop (best practice )
int x;
// in parens (starting point; ending point; count by)
for (x=0;x<10;x++){
    printf("%d\n",x);

}
int m;
for (m=0;m<mlength;m++){
    
};
//How do you make while loops in C?"
int w = 0;
while(w<100)
    printf ("%d\n",w);
    w+=10;

    return 0;
}